README.md

```diff
@@ -14,7 +14,20 @@
 
 ##Installation Instructions
 
-    composer require silverstripe/wordpressimport 0.3.*
+    composer require thiagovandieten/export-wordpresstosilverstripe *
```

Comment:
Agreed, but silverstripe/wordpress doesn't show up on http://addons.silverstripe.org/add-ons
I can't find it with composer either (don't know why).
The only module I could find and install was thiagovandieten/export-wordpresstosilverstripe, so I thought that would be the "official" now...